extensions-contrib/druid-rocketmq/src/main/java/org/apache/druid/firehose/rocketmq/RocketMQFirehoseFactory.java

```diff
@@ -404,6 +404,7 @@ public void putRequest(final DruidPullRequest request)
         this.requestsWrite.add(request);
         if (!hasNotified) {
           hasNotified = true;
+          // No need to use notifyAll here since extended class com.alibaba.rocketmq.common.ServiceThread handles it
```

Comment:
Probably better:

> No need to use notifyAll here; following convention of parent class com.alibaba.rocketmq.common.ServiceThread

Reply:
I've updated my PR 👍

Reply:
@gianm is current changes OK for this PR?